src/random/urandom.nim

```diff
@@ -65,33 +65,35 @@ template urandomImpl(): stmt {.immediate.} =
   when defined(windows):
     if cryptProv == 0:
       urandomInit()
-    
+
     let success = CryptGenRandom(cryptProv, DWORD(size), addr result[0])
     if success == 0:
       raise newException(OSError, "Call to CryptGenRandom failed")
-  
+
```

Comment:
Please don't change whitespace. Some text editors can fix whitespace only on modified lines. Otherwise you can do `git add --patch` to add only needed changes.